packages/runtime/datastore-definitions/src/test/types/jsonable.ts

```diff
@@ -176,6 +184,7 @@ foo(a14);
 
 // test class with function
 class bar {
+	// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
We can just add the return type instead, no?

Reply:
Oh, yeah, I suppose we can. I was trying not to be too invasive here since they are testing type concepts. But adding it here makes no type-wise difference. Will update.